src/app/forms/components/required-standard-select/required-standard-select.component.ts

```diff
@@ -14,13 +18,13 @@ import { Subject, takeUntil } from 'rxjs';
 })
 export class RequiredStandardSelectComponent implements OnInit, OnDestroy {
 
-  requiredStandards?: SectionIVA[];
+  requiredStandards?: RequiredStandardTaxonomySection[];
```

Comment:
Are these changes because of the package upgrade? was VTM not already using this version of the IVA contracts?

Reply:
No it wasn't updated. These are just renaming imports basically, so there is no change happening functionally at all